youtube_dl/postprocessor/ffmpeg.py

```diff
@@ -429,20 +431,43 @@ def add(meta_list, info_list=None):
 
         filename = info['filepath']
         temp_filename = prepend_extension(filename, 'temp')
+        in_filenames = [filename]
+        tmp_filenames = []
+
+        options = ['-map_metadata', '1']
```

Comment:
ahhh, as i created the patch a long time, i forget that the `1` is a index of input file and it doesn't map metadata from the original video file, will fix it.